app/graphql/mutations/transformers/panoptes_classification.rb

```diff
@@ -0,0 +1,66 @@
+module Transformers
+  class PanoptesClassification
+    attr_accessor :payload
+
+    def initialize(payload)
+      @payload = payload
+    end
+
+    def transform
+      {
+        event_id:        id,
+        event_type:      type,
+        event_source:    source,
+        event_time:      time,
+        project_id:      project,
+        workflow_id:     workflow,
+        user_id:         user,
+        data:            remaining_data,
+        session_time:    session_time
+      }
+    end
+
+    private
+    def id
+      payload.dig("data", "id")
+    end
+
+    def type
+      payload.dig("type")
+    end
+
+    def source
+      payload.dig("source")
+    end
+
+    def time
+      DateTime.parse(payload.dig("data", "metadata", "finished_at"))
```

Comment:
Yeh that what I was thinking. Since I am already using started_at and finished_at for session time, it makes sense to use the most accurate timestamp for when they did the classification